enterprise/kernel/src/main/java/org/neo4j/kernel/impl/store/format/highlimit/v30/HighLimitV3_0.java

```diff
@@ -55,7 +55,8 @@
 
     public HighLimitV3_0()
     {
-        super( STORE_VERSION, 7, Capability.DENSE_NODES, Capability.SCHEMA, Capability.LUCENE_5 );
+        super( STORE_VERSION, 7, Capability.DENSE_NODES, Capability.SCHEMA, Capability.LUCENE_5,
+                Capability.FIXED_REFERENCE );
```

Comment:
Correct, if we need to perform store migration only when store capabilities changed - we need new capability for newly introduced fixed references - to avoid migration between high limit 3.0 and high limit 3.1, but still have migration from standard format.